proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+
+    2.   microsoft [author]
+        Certificate fingerprint(s):
+            SHA256 - 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
+            SHA256 - AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27
+
+    3.   myUntrustedAuthorSignature [author]
+        Certificate fingerprint(s):
+            [U] SHA256 - 518F9CF082C0872025EFB2587B6A6AB198208F63EA58DD54D2B9FF6735CA4434
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `> dotnet nuget trust sync`
+
+```
+Requests the latest list of certificates used in a currently trusted repository to update the the existing certificate list in the trusted signer.
+
+Note: This gesture will delete the current list of certificates and replace them with an up-to-date list from the repository.
+
+USAGE:
+    dotnet nuget trust sync [OPTIONS] <name>
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `> dotnet nuget trust remove`
+
+```
+Removes any trusted signers that match the given name.
+
+USAGE:
+    dotnet nuget trust remove [OPTIONS] <name>
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `> dotnet nuget trust author`
+
+```
+Adds a trusted signer with the given name to the config, based on the author signature(s) of one or more packages. If <name> already exists in the configuration, the signature(s) will be appended.
+
+USAGE:
+    dotnet nuget trust author [OPTIONS] <name> <package>...
+
+EXAMPLE:
```

Comment:
I do not see this example adding much.

Reply:
To me `<name>` and `<package>` are very abstract terms. From this command alone, it's unclear how `<name>` will be used. Also, it's also unclear that `<package>` should be a path to a .nupkg file. The example does help clarify some of these questions.

Is it possible to add descriptions for the command arguments? Maybe that'd be better than an example.